src/e2e/puppeteer/__tests__/caret.ts

```diff
@@ -172,7 +174,7 @@ describe('all platforms', () => {
 describe('mobile only', () => {
   beforeEach(async () => {
     await emulate(KnownDevices['iPhone 11'])
-  }, 5000)
+  }, 20000)
```

Comment:
Did this timeout need to increase from 5s to 20s because of the additional bundle size with the animations? I'm guessing this is unavoidable, but just wanted to ask because we'd like to keep the test execution as brief as possible.

Reply:
The temporary increase in Puppeteer test timeout to 20000ms was solely for debugging purposes. I’ve now restored it to the original setting.